### DISCUSSION TITLE
pen movement in lazer is inverted

### DISCUSSION THREAD

**Discussion by adrianc420** [ppy/osu#25591] [Q&A]
Problem details: i dont want to use the internal tablet drivers so i disabled them and also disabled raw input, however now when i move my pen up on my tablet its inverted in game, only happens in lazer. and this isnt happening with my mouse. the drivers i use are opentabletdriver.

osu! lazer latest version

**Comment by nick22985** [comment]
For anyone wondering try disabling "High precision mouse" in the input settings
<img width="1124" height="570" alt="swappy-20250927_234021" src="https://github.com/user-attachments/assets/2c9a859e-730a-4823-98cb-6684cf184251" />
